test/ctia/http/routes/incident_test.clj

```diff
@@ -22,6 +25,52 @@
 
 (use-fixtures :each whoami-helpers/fixture-reset-state)
 
+(defn partial-operations-tests [incident-id incident]
+  (testing "POST /ctia/incident/:id/status Open"
+    (let [new-status {:status "Open"}
+          response (post (str "ctia/incident/" (:short-id incident-id) "/status")
+                         :body new-status
+                         :headers {"Authorization" "45c1f5e3f05d0"})
+          updated-incident (:parsed-body response)]
+      (is (= 200 (:status response)))
+      (is (= "Open" (:status updated-incident)))
+      (is (get-in updated-incident [:incident_time :opened]))
+
+      (is (t/within?
+           (t/interval (t/minus (t/now) (t/minutes 10))
+                       (t/plus (t/now) (t/minutes 10)))
```

Comment:
Could you consider using a `with-redef` for `now` instead of using magic numbers here? I think there should be a fixture for that.